30 Dataview Resources/33 Use Cases/Learn the Basics.md

```diff
@@ -0,0 +1,45 @@
+# Learn the Basics
+
+This page shall guide you if you are a newcomer to the power of Dataview. **Before you start out**, make sure you made yourself familiar with the [official Documentation](https://blacksmithgu.github.io/obsidian-dataview/). While the documentation is not the most intuitive guide, it's important to get a grasp of the general idea and functionality to enable you writing your own queries. 
+
+There are some [[Dataview Documentation and Resources|further reading]] you can dive into, but that's not necessary to get started. If you stumble across something, maybe the [[FAQ]] can help out.
+
+## Understanding meta data on notes
+
+Before you can start querying data from your vault, you need [**meta data**](https://blacksmithgu.github.io/obsidian-dataview/data-annotation/) to query. While there are some information already available on your notes, to leverage the full power of dataview, you'll need to add your own meta data via the frontmatter or inline fields.
```

Comment:
[...] While there is already some basic metadata available on your notes, to leverage [...]

Question: Do we want to refer/explain what metadata dataview automatically generates for you?